identity/liquid-templates/template.html

```diff
@@ -64,81 +64,63 @@
         padding-left: 0;
         padding-right: 0;
       }
+      ._prompt-box-outer * {
+        text-align: left!important;
+      }
+      ._prompt-box-outer button {
+        width: auto!important;
+      }
+      ._prompt-box-outer div {
+        padding-left: 0!important;
+        min-height: auto!important;
+      }
+      ._prompt-box-outer .success-email {
+        display: none;
+      }
       /* end of hacky overrides of prompt styling */
       body {
         margin: 0;
+        background: #ffffff;
+        color: #121212;
         font-family: "helvetica neue", helvetica, arial, sans-serif;
-        /* TODO load webfont? */
         font-size: 16px;
-        color: #121212;
         line-height: 1.5;
       }
-      .background {
-          background: #ffffff;
-        }
-      @media (min-width: 920px) {
-        .background {
-          background: linear-gradient(90deg, #e8e8e8 50%, #ffffff 50%);
+      @media (min-width: 803px) {
+        body {
+          background: #f0ede3;
         }
       }
-      .logo {
-        display: inline-block;
-        margin-top: 50px;
-        margin-left: 50%;
-        transform: translateX(-50%);
-      }
-      .container {
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-      }
-      @media (min-width: 920px) {
-        .container {
-          flex-direction: row-reverse;
-        }
+      .header {
+        background: #ffffff;
+        border-bottom: 1px solid #d9d6ce;
+        height: 85px;
+        padding-top: 23px;
```

Comment:
Again, some funny values?

Reply:
This places the logo in the same position as the main site